src/explanations/units_variables.yaml

```diff
@@ -0,0 +1,171 @@
+- title: Units
+  shortTitle: Units
+  key: "units"
+  details: |
+    - Ether Units
+      While using Ethereum, we may encounter different units, such as Wei and Gwei. In Solidity, we can use wei, gwei or ether to specify a subdenomination of Ether. 
+      Wei is the smallest unit of measurement. One Wei is worth 0.000000000000000001 Ether (or 10-18). Because it cannot deal with decimal numbers, the EVM (Ethereum Virtual Machine) employs Wei as its base unit. Gwei, another common unit, is 1000000000 Wei, or 0.000000001 (or 10-9) Ether. 
+      
+      - `1 Wei = 1;`
```

Comment:
Improve the alignments as half of them are not aligned correctly

Reply:
sorry that was not visible in my local IDE at all.